cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -133,6 +133,8 @@ func Run(s *options.CMServer) error {
 	// Override kubeconfig qps/burst settings from flags
 	kubeconfig.QPS = s.KubeAPIQPS
 	kubeconfig.Burst = int(s.KubeAPIBurst)
+
+	// Create a Rest Client Object used to access the Kubernetes api Server provides API services
```

Comment:
sorry for the delay in responding, I am still catching up from being OOO.

Unfortunately I think this comment is hard to understand. Also this line makes a client set, not a RESTClient. (Yes, there is an embedded RESTClient, but that is an implementation detail and it's not good to mention it here.)

If I were to add a comment here, I would say something like "Add the controller-manager user agent to all requests we make with this client, to make it easy to tell what is responsible for the requests when you view the apiserver logs".

But generally I think comments should tell people *why* you are doing something, not *what* is being done. For the latter, people should just read the code. Comments like that quickly become out of date.